orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/tasks/SavePipelineTaskSpec.groovy

```diff
@@ -126,4 +126,67 @@ class SavePipelineTaskSpec extends Specification {
     then:
     receivedIndex == newIndex
   }
+
+  def "should update runAsUser with service account in each trigger where it is not set "() {
+    given:
+    String runAsUser
+    def expectedRunAsUser = 'my-pipeline@managed-service-account'
+    def pipeline = [
+      application: 'orca',
+      name: 'my pipeline',
+      stages: [],
+      triggers: [
+        [
+          type: 'cron',
+          enabled: true
+        ]
+      ]
+    ]
+    def stage = new Stage(Execution.newPipeline("orca"), "whatever", [
```

Comment:
Should be able to use the `stage` builder for tests here:

```
def stage = stage {
  context = [
    pipeline: Base64.encoder.encodeToString(...),
   "pipeline.serviceAccount": expectedRunAsUser
  ]
}
```
Stage builder: `import static com.netflix.spinnaker.orca.test.model.ExecutionBuilder.stage`.

Reply:
Hmm I guess other tests in this spec don't use that builder

Reply:
Good to know since this method seems to be deprecated. Changed it in the other file but leaving it as is here for consistency.